.gitignore

```diff
@@ -1,3 +1,6 @@
+.venv/
+
+
```

Comment:
I would prefer if the two extra empty lines were removed.

Reply:
Will fix. Thank you.

Reply:
Fixed